src/App.tsx

```diff
@@ -1,13 +1,47 @@
 import { useState } from "react";
 import { useForm } from "react-hook-form";
-import { Button } from "@atomicjolt/atomic-elements";
 import { Form, FormProvider } from "../lib";
 import "./App.css";
 
+interface Fields {
+  name: string;
+  description: string;
+  age: number | null;
+  ethnicity: {
+    choice?: string | null;
+    custom?: string | null;
+  };
+  address?: string;
+  number: string | null;
+  notifications: boolean;
+  hearAbout: string[];
+  notificationFrequency: string | null;
+  consent: boolean;
+}
+
+const defaultValues: Fields = {
+  name: "Sean",
```

Comment:
Do you want to leave Sean in here as a default?

Reply:
Everything in src is just a demo app, so it doesn't particularly matter what the defaults are :)